Sources/CvPlayer.cpp

```diff
@@ -6401,7 +6400,7 @@ void CvPlayer::receiveGoody(CvPlot* pPlot, GoodyTypes eGoody, CvUnit* pUnit)
 			int iBestValue = 0;
 			pBestPlot = NULL;
 
-			foreach_(CvPlot* pLoopPlot, CvPlot::rect(pPlot->getX(), pPlot->getY(), iOffset, iOffset)
+			foreach_(CvPlot* pLoopPlot, pPlot->rect(iOffset, iOffset)
 			| filtered(!CvPlot::fn::isRevealed(getTeam(), false)))
 			{
```

Comment:
It would be helpful when using these binds that you add some comment with what is happening, so other modders can follow the code down the line

Reply:
There is pinned message in cpp channel. I couldn't explain better then bill or google.

Reply:
Sure i know that now, but new modders that come to later maybe wont